### DISCUSSION TITLE
Fody removed?

### DISCUSSION THREAD

**Discussion by MartinFayRAX** [reactiveui/ReactiveUI#3809] [Q&A] [answered]
I've just been checking package updates and noticed that ReactiveUI.Fody is no longer supported.

(a) Can anyone give me any guidance on the intended replacement instead of using the Fody [Reactive] and [ObservableAsProperty] attributes? i.e. is there anything other than "old school" manual implementation?

(b) I'd be curious to know any background leading to this choice to remove functionality?

Thank you! Martin

**Comment by ChrisPulman** [comment, marked as answer]
Hi it's not that it's not supported we haven't made any actual updates to the Fody package through many releases so we removed it until we can create a new Source Generator version

**Reply by MartinFayRAX** [reply]
Thank you Chris! I was wondering whether Source Generator would have an impact on Fody use when I noticed this...

**Reply by ChrisPulman** [reply]
Just incase you hadn't seen, there is now ReactiveUI.SourceGenerators available which has far more functionality available than the ReactiveUI.Fody options.
